### DISCUSSION TITLE
Cannot get my remote to work

### DISCUSSION THREAD

**Discussion by monisallam** [GerryDazoo/Slinger#372] [Q&A]
Is it still expected to see an 'RT' button on the default remote in the latest repo? I do not see it, my remote isnt working at all and I am trying to run through the debugging steps... hopefully this isnt a repeat discussion I tried to check

**Comment by GerryDazoo** [comment]
The RT command was deprecated once we figured out it was the same number as VideoSource.

**Comment by monisallam** [comment]
Okay got it - so I think its safe to assume I have the wrong mappings in the remote to the IR codes - I tried to use the Fios one (I have fios) in the CustomRemotes folder, but it didn't seem to work. I saw another one in the discussions here that Ill try out.

Thank you for doing this whole thing @GerryDazoo - its awesome, and was really fun to try to implement.

**Reply by socelillyhammer** [reply]
@monisallam any luck on the remote?  I also have Fios One.  When my SB was working with my desktop/webplayer the remote on the webplayer was working.  So I assume (becuase I did not reset my box) the remote should work.  But no luck on my end. HBU?

**Comment by monisallam** [comment]
@socelillyhammer same exact set up here - Fios One, didn't reset the slingbox and no luck. I tried every button to see if anything will change the state of the TV but nothing would.

**Reply by socelillyhammer** [reply]
@monisallam cool. Ive been searching the chats for solutions.  Will keep you posted if I find anything.

**Reply by monisallam** [reply]
@socelillyhammer thanks! Ill do the same - I am wondering if the IR codes are just totally different than prior fios remotes or other vendor remotes - like not decimal digits in the 1-50 range but I have no clue.

**Reply by GerryDazoo** [reply]
You have to be watching a stream with the server for the remote to work

**Reply by socelillyhammer** [reply]
no clue.  I have my IR reciever attached to my SB500 as well. But I think the coding is different.  Im trying to scour the internet to see if anyone may have decrypted or figured out the coding to the p265 or p285 the older remotes but it was the ones that SB used for vz.

Im not savy enough to figure out how, if possible, to link into the bluetooth of the remote and download/extract the coding but I think that would yield a solution if that can be done.

**Reply by socelillyhammer** [reply]
at this point there is a SB alternative but the box costs 2k plus annual maintenance of $300 a year.

**Reply by socelillyhammer** [reply]
@GerryDazoo I am watching on VLC and have the remote on my firefox open at the same time.  Its not working

**Reply by monisallam** [reply]
@GerryDazoo same^ - stream is open, I can see the codes being sent in the log, just no change of state

**Reply by monisallam** [reply]
@socelillyhammer what do you mean you have an IR receiver attached to the SB? Isnt the IR receiver inside the FIOS One box?

**Reply by JoeM2016** [reply]
@socelillyhammer There is a discussion in the Tivo Community Forum (www.tivocommunity.com) about sb's being discontinued. In addition to slinger being referenced there is a description of a system that is about $250 with no annual costs. Start reading the posts from the second last page.

**Reply by socelillyhammer** [reply]
@JoeM2016 thank you but its not what im looking for.  SB had a direct live stream which is why im reading through this entire forum and learning to figure out how to work the code.  what they are talking about there is a different animal this ramio item

**Reply by socelillyhammer** [reply]
@JoeM2016 there is a company that has a box and costs $2k for a live streaming box and $300 a year maintenance. But Im trying to use that as a last resort

**Reply by JoeM2016** [reply]
The system (a hdmi to IP encoder) on the Tivo forum is like a sb where it has hdmi input and output connections and uses an ir blaster for remote control. The encoder is then connected to your router.

**Reply by Toei79** [reply]
whats your issue here a code for remote? curious if logitech dont have it, those guys are always very polite and they used to sell amazing universal remotes controllers, i still using those. barely but still.

**Reply by monisallam** [reply]
@Toei79 the issue is the codes that Fios One uses to change the state of the TV - they dont seem to be the same at all as any other remote. I dont think the remote is a logitech one.

**Reply by socelillyhammer** [reply]
@monisallam in looking at all the coding for the other remotes on the javascript they all seem to read pretty much the same.  Ever since fios came out with the fios one things have gotten screwy. But SB did support the new devices and as long as you found your remote on SB when it was up and running, the remote did work.

**Reply by monisallam** [reply]
@socelillyhammer yep - I am inclined to think that the codes are no longer decimals from 1-50ish given that not a single button has affected anything on my end - I scoured online documentation but still cant come up with anything. I found [this](http://pejaver.com/Projects/IRRemote.htm) , which has a bunch of technical stuff but it has different mappings between buttons and codes but those didn't work either. 

One last possible thing to do is write a script to loop through the possible digits in a 0x0000 pattern but that just seems insane to me and potentially no even going to work.

**Comment by socelillyhammer** [comment]
@monisallam Yes.  SB500 also came with an IR Emitter Cable as an optional
device. So I threw that on just in case.

**Comment by socelillyhammer** [comment]
If this is the coding for the remote, this is awesome.  script to loop - I
don't understand what that means nor would I know how to write any of it.
But I'd be happy to sponsor and pay someone for the work if it will get
the remote to work.

**Reply by monisallam** [reply]
@socelillyhammer unfortunately I don't think they work - in terms of the script its just like a tiny automation/program that would theoretically try all possible codes - I could write it free of charge lol but it would be kind of a pain in the butt because it would be 16x16x16x16=65536 possible codes and I would have to sit in front of the TV to see which ones worked, plus there is a timing issue where after each code is entered, you have to wait because of potential lag.... basically huge pain in the ass. Like the code itself is not the hard part if that makes sense

**Reply by socelillyhammer** [reply]
Does not make sense.  If there is a way we can test out one button to see if the code works, is there any possible way you can show me what do to do and I can just do a "copy paste" over the weekend.  i.e. you create the first store (asusme like 1 hour to build), and if it works, you just tell me what I need to do to build the other 400 stores becuase the model is already in place (i.e. all the other buttons and spend the other 50 hours).  Then I can sit and put the time into finding the codes or lagging or whatever.  Team work on this. and when its done I'll send you the data so you can confirm it all to finish up the project.  What do you think?

**Reply by socelillyhammer** [reply]
@monisallam Here is a thought which might make things easier and faster all around. verizon myfios app on android has a remote built in for the fios one and when we login to it if we are on our home network we can use that remote to change the channels.

Theory: do you think someone can open the apk file and extract the remote and the codes and we decipher that way?

**Reply by monisallam** [reply]
@socelillyhammer the android idea seems like a good one, at least a way better one than mine.

**Comment by mdehrlich** [comment]
Looking at remote on Amazon, it appears this is a bluetooth device.  Not clear if the Fios one box has ir control.  Slingbox only does ir.  Did your slingbox ever control your FIOS one STB?

**Comment by socelillyhammer** [comment]
Yes the SB did control the STB with the p265 remote or a version of it.
The STB does have an ir receiver I just think it's a coding issue. My SB
settings still has the correct receiver settings and was working with my
STB on Nov 9.  But idk how to deal with the remote coding.

**Comment by monisallam** [comment]
@socelillyhammer are you sure the STB has an IR receiver? The slingbox desktop remote stopped working for me after the upgrade to the new Fios One STB. (pre slingbox service shutdown)

**Comment by socelillyhammer** [comment]
Yup. Check this out here as it uses both.
https://forums.verizon.com/t5/fios-tv-technical-assistance/ir-extender-flasher-with-fios-tv-one-remotes-has-anyone-had/td-p/891178

**Comment by socelillyhammer** [comment]
@monisallam  I found this
http://www.hifi-remote.com/wiki/index.php/How_to_Install_a_Custom_Remote_onto_the_Slingbox

But would $300 be satisfactory to create a working remote for the FIOS One Box  IPC1100 VMS4100 etc.
